### PR TITLE
Expand arrays returned by `Get-TenableGroup*`

### PR DESCRIPTION
**Pull request by RobFaie**

### CODE REVIEW

File: functions/Get-Group.ps1 [hunk 1]
```diff
@@ -24,5 +24,12 @@ function Get-Group {
     )
 
     $path = '/groups'
-    Invoke-Method -Context $Context -Path $path -Verbose:$VerbosePreference
+    $result = Invoke-Method -Context $Context -Path $path -Verbose:$VerbosePreference
+    if ($result) {
+        if ($result | Get-Member -Name groups) {
+            $result | ForEach-Object groups
```
Comment (mderriey) [inline functions/Get-Group.ps1:30]:
```suggestion
            $result.groups
```

File: functions/Get-GroupMember.ps1 [hunk 2]
```diff
@@ -31,5 +31,12 @@ function Get-GroupMember {
 
     $path = "/groups/$GroupId/users"
 
-    Invoke-Method -Context $Context -Path $path -Verbose:$VerbosePreference
+    $result = Invoke-Method -Context $Context -Path $path -Verbose:$VerbosePreference
+    if ($result) {
+        if ($result | Get-Member -Name users) {
+            $result | ForEach-Object users
```
Comment (mderriey) [inline functions/Get-GroupMember.ps1:37]:
```suggestion
            $result.users
```

**Review by mderriey — APPROVED** [review]
LGTM 👍

Curious as to whether the dot notation is different from the `ForEach-Object` one.

File: functions/Get-Group.ps1 (hunk 1 above)
Comment (RobFaie) [reply functions/Get-Group.ps1:30]: Use of `ForEach-Object` is a safer option. In circumstances where no data is returned by `Invoke-Method` the dot notation causes an exception while the `ForEach-Object` does not.

File: functions/Get-GroupMember.ps1 (hunk 2 above)
Comment (RobFaie) [reply functions/Get-GroupMember.ps1:37]: Use of ForEach-Object is a safer option. In circumstances where no data is returned by Invoke-Method the dot notation causes an exception while the ForEach-Object does not.

**Pull request merged**